docs/core/resilience/snippets/resilience/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Polly;
+using Polly.CircuitBreaker;
+using Polly.Registry;
+using Polly.Retry;
+
+var services = new ServiceCollection();
+
+const string key = "Retry-CircuitBreaker-Timeout";
+
+services.AddResiliencePipeline(key, builder =>
+{
+    // See: https://www.pollydocs.org/strategies/retry.html
+    builder.AddRetry(new RetryStrategyOptions());
+
+    // See: https://www.pollydocs.org/strategies/circuit-breaker.html
+    builder.AddCircuitBreaker(new CircuitBreakerStrategyOptions());
+
+    // See: https://www.pollydocs.org/strategies/timeout.html
+    builder.AddTimeout(TimeSpan.FromSeconds(1.5));
+});
+
+services.AddResilienceEnrichment();
+services.AddExceptionSummarizer(); // TODO: remove once bug is fixed.
```

Comment:
Maybe link to the GitHub issue?

Reply:
I should be able to remove this... we shouldn't need this at all soon.